src/sections/Learn/Book-single/BookSingle.style.js

```diff
@@ -27,6 +27,12 @@ const BookSinglePageWrapper = styled.div`
 		}
 		
 	}
+
+	@media only screen and (max-width: 480px) {
```

Comment:
It needs to start from 500 instead of 480, because I can see the overflow starting from 500px itself.

Reply:
Also, make the text in the row center-aligned

Reply:
Okay! I'll make those changes.

Reply:
I didn't quite get which row is meant to be center-aligned other than the ones that already are. 
Are you referring to the text inside select block?

Reply:
The table rows

Reply:
Hey @Nikhil-Ladha ,
 Is this Okay? 
 
![image](https://github.com/layer5io/layer5/assets/75237697/f298224d-e9fd-4804-b631-9ce444431c8a)

Reply:
I think This Would be Better:
![image](https://github.com/layer5io/layer5/assets/75237697/2793d991-5fc9-4af1-a40f-ecac3919e441)

Reply:
Let's go with the former, where all the content is center-aligned.

Reply:
@Nikhil-Ladha, I have made those changes and added a commit for the same.

Reply:
When I said "former", I meant both both the columns text should be center-aligned.
Also, the alignment is required for all screen resolution not only for mobile view.

Reply:
@Nikhil-Ladha I must have messed that up right before sending the commit.
Fixed now, Also made the alignment for all resolutions.